modules/security-audit-log-filtering.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * security/audit-log-view.adoc
+
+[id="security-audit-log-basic-filtering_{context}"]
+= Filtering audit logs
+
+You can use `jq` or another JSON parsing tool to filter the API server audit logs.
+
+[NOTE]
+====
+The amount of information logged to the API server audit logs is controlled by the audit log policy that is set.
+====
+
+The following procedure provides examples of using `jq` to filter OpenShift API server audit logs on node `node-1.example.com`. See the link:https://stedolan.github.io/jq/manual/[jq Manual] for detailed information on using `jq`.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed `jq`.
+
+.Procedure
+
+* Filter audit logs by user:
++
+[source,terminal]
+----
+$ oc adm node-logs node-1.example.com  \
+  --path=openshift-apiserver/audit.log \
+  | jq 'select(.user.username == "myusername")'
+----
+
+* Filter audit logs by user agent:
++
+[source,terminal]
+----
+$ oc adm node-logs node-1.example.com  \
+  --path=openshift-apiserver/audit.log \
+  | jq 'select(.userAgent == "cluster-version-operator/v0.0.0 (linux/amd64) kubernetes/$Format")'
+----
+
+* Filter audit logs by a certain API version and only output the user agent:
++
+[source,terminal]
+----
+$ oc adm node-logs node-1.example.com  \
+  --path=openshift-apiserver/audit.log \
+  | jq 'select(.requestURI | startswith("/apis/apiextensions.k8s.io/v1beta1")) | .userAgent'
```

Comment:
I tried this example with 4.7, didn't get any results, so I tried another example replaced by the following, get some results.
`jq 'select(.requestURI | startswith("/apis/apps.openshift.io/v1")) | .userAgent'`
I think it is better to use above as example.

Reply:
`kube-apiserver/audit.log` will have those events

Reply:
Thanks @wangke19! I checked with @sttts because he gave me the idea to use v1beta1 in the example. Turns out it needed to be for the kube-apiserver to return results. So I've updated the example and adjusted other wording a bit.

I also addressed your other comment. Let me know if this looks okay now or if you have any more feedback.

Reply:
@bergerhoffer, no more feedback.